chemprop/data/scaffold.py

```diff
@@ -120,7 +120,7 @@ def scaffold_split(data: MoleculeDataset,
                      f'val scaffolds = {val_scaffold_count:,} | '
                      f'test scaffolds = {test_scaffold_count:,}')
 
-    if logger is not None:
+    if logger is not None and data[0].atom_targets is None and data[0].bond_targets is None:
```

Comment:
Triggering logic off of `data[0].atom_targets is None` seems a bit complex for something you may want to check in lots of places. As long as we're not doing mixed molecule/atom targeting, it probably would make more sense to have `atom_bond_target: bool` as a dataset property.